grants/equity_brix/data.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Likewise, Equity Brix metadata contains Ramp Network info. Please check and update as you see fit.